releases/release-5.1.0.md

```diff
@@ -0,0 +1,344 @@
+| TiKV configuration file | [`sink-memory-quota`](/tikv-configuration-file.md#sink-memory-quota)  | Newly added  | Sets the upper limit of memory usage by TiCDC data change events. The default value is `512MB`. |
+| TiKV configuration file | [`incremental-scan-threads`](/tikv-configuration-file.md#incremental-scan-threads)  | Newly added  | Sets the number of threads for the task of incrementally scanning historical data. The default value is `4`, which means there are four threads for the task.  |
+| TiKV configuration file | [`incremental-scan-concurrency`](/tikv-configuration-file.md#incremental-scan-concurrency)  | Newly added  | Sets the maximum number of concurrent executions for the tasks of incrementally scanning historical data. The default value is `6`, which means that six tasks can be concurrently executed at most. |
+| TiKV configuration file | [`soft-pending-compaction-bytes-limit`](/tikv-configuration-file.md#soft-pending-compaction-bytes-limit)  | Modified  | The soft limit on the pending compaction bytes. The default value is changed from `"64GB"` to `"192GB"`. |
+| TiKV configuration file | [`storage.io-rate-limit`](/tikv-configuration-file.md#storageio-rate-limit)  | Newly added  | Controls the I/O rate of TiKV writes. The default value of `storage.io-rate-limit.max-bytes-per-sec` is `"0MB"`. |
+| TiKV configuration file | [`resolved-ts.enable`](/tikv-configuration-file.md#enable)  | Newly added  | Determines whether to maintain the `resolved-ts` for all Region leaders. The default value is `true`. |
+| TiKV configuration file | [`resolved-ts.advance-ts-interval`](/tikv-configuration-file.md#advance-ts-interval)  | Newly added  | The interval at which the `resolved-ts` is forwarded. The default value is `"1s"`. You can change the value dynamically. |
+| TiKV configuration file | [`resolved-ts.scan-lock-pool-size`](/tikv-configuration-file.md#scan-lock-pool-size)  | Newly added  | The number of threads that TiKV uses to scan the MVCC (multi-version concurrency control) lock data when initializing the `resolved-ts`. The default value is `2`. |
+
+### Others
+
+- Upgrade the Go compiler version of TiDB from go1.13.7 to go1.16.4, which improves the TiDB performance. If you are a TiDB developer, upgrade your Go compiler version to ensure a smooth compilation. 
+- Avoid creating tables with clustered indexes in the cluster that uses TiDB Binlog during the TiDB rolling upgrade.
+- Avoid executing statements like `alter table ... modify column` or `alter table ... change column` during the TiDB rolling upgrade.
+- Since v5.1, setting the replica of system tables, when building TiFlash replicas for each table, is no longer supported. Before upgrading the cluster, you need to clear the relevant system table replicas; otherwise, the upgrade will fail..
```

Comment:
```suggestion
- Since v5.1, setting the replica of system tables, when building TiFlash replicas for each table, is no longer supported. Before upgrading the cluster, you need to clear the relevant system table replicas; otherwise, the upgrade will fail.
```